ILSwiss/rlkit/torch/algorithms/sac/conditional_sac_alpha.py

```diff
@@ -0,0 +1,256 @@
+from collections import OrderedDict
+
+import numpy as np
+import torch
+import torch.optim as optim
+
+import rlkit.torch.utils.pytorch_util as ptu
+from rlkit.core.trainer import Trainer
+from rlkit.core.eval_util import create_stats_ordered_dict
+
+
+class ConditionalSoftActorCritic(Trainer):
```

Comment:
Can this class be inherited from SoftActorCritic?

Reply:
fixed